### PR TITLE
fix std::put_time error on travis

### PR DESCRIPTION
**Pull request by samantharitter**

### CODE REVIEW

File: src/bsoncxx/test/oid.cpp [hunk 1]
```diff
@@ -48,15 +51,21 @@ parsed_oid parse_oid(const oid& oid) {
...
 #endif
 
     return parsed;
 }
 
 void compare_string(const std::time_t& t, std::string time) {
-    std::stringstream time_stream;
-    time_stream << std::put_time(std::gmtime(&t), "%b %e, %Y %H:%M:%S UTC");
-    REQUIRE(time_stream.str() == time);
+    // std::stringstream time_stream;
```
Comment (kevinAlbs) [inline src/bsoncxx/test/oid.cpp:61]: Remove comments.

File: src/bsoncxx/test/oid.cpp [hunk 2]
```diff
@@ -48,15 +51,21 @@ parsed_oid parse_oid(const oid& oid) {
...
+    char time_str[48];
+    if (0 == (strftime(time_str, sizeof(time_str), "%b %e, %Y %H:%M:%S UTC", std::gmtime(&t)))) {
```
Comment (kevinAlbs) [inline src/bsoncxx/test/oid.cpp:63]: Perhaps `REQUIRE (0 != strftime(...))`?

**Review by kevinAlbs — APPROVED** [review]
Thanks for looking into this! LGTM with some minor cleanup.

File: src/bsoncxx/test/oid.cpp (hunk 2 above)
Comment (samantharitter) [reply src/bsoncxx/test/oid.cpp:63]: Yes this would be better!  Done.

File: src/bsoncxx/test/oid.cpp (hunk 1 above)
Comment (samantharitter) [reply src/bsoncxx/test/oid.cpp:61]: Done.

**Review by kevinAlbs — APPROVED** [review]

**Pull request merged**